SPECS/mstflint/mstflint.spec

```diff
@@ -1,15 +1,14 @@
-Summary:        Mellanox firmware burning tool
+Summary:        Mellanox firmware burning application
 Name:           mstflint
-Version:        4.21.0
-Release:        4%{?dist}
+Version:        4.26.0
+Release:        1%{?dist}
 License:        GPLv2 OR BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
 URL:            https://github.com/Mellanox/%{name}
-Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
-Patch4:         add-default-link-flags-for-shared-libraries.patch
-Patch6:         replace-mlxfwreset-with-mstfwreset-in-mstflint-message.patch
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-%{release}/%{name}-%{version}-%{release}.tar.gz
```

Comment:
It looks like `-1` is part of the upstream URI to this source archive. Note that we need the ability to revise this .spec even if that upstream version doesn't change -- at which point the release would become `2%{?dist}` and the URI won't resolve, no?

Reply:
yeah, make sense, seems I tried to follow their logic, but is not applicable on our side. Thanks for pointing this out.